lib/package.gd

```diff
@@ -691,6 +691,31 @@ DeclareGlobalFunction( "LoadPackageDocumentation" );
 ##
 ##  <Description>
 ##  loads the &GAP; package with name <A>name</A>.
+## <P/>
+##  As an example, the following loads the &GAP; subgroups package
+## <Package>Sonata</Package>:
+##  <P/>
+##  <Log><![CDATA[
+##  gap> LoadPackage( "Sonata" );
```

Comment:
The authors write the name of the package in UPPERCASE, but `LoadPackage` accepts any case. I suggest to use `LoadPackage( "sonata" );` as most commonly used example.